.github/workflows/tox-pytest.yml

```diff
@@ -1,7 +1,13 @@
 ---
 name: tox-pytest
 
-on: [pull_request]
+on:
+  pull_request:
```

Comment:
Might we want to add a `workflow_dispatch` here so someone can manually kick off the integration tests once without needing to change the status of the PR?

Reply:
Maybe. I'm not sure how the workflow_dispatch event will interact with the `if:` expression, since then presumably `pull_request.event` won't be defined.

I'm inclined to keep this change as-is and not deal with that, since it's pretty easy to change the status of the PR to get integration tests running if one wants them.